src/solvers/applyexp.jl

```diff
@@ -0,0 +1,121 @@
+using Printf: @printf
+
+@kwdef mutable struct ApplyExpProblem{State}
+  operator
+  state::State
+  current_exponent::Number = 0.0
+end
+
+ITensorNetworks.state(A::ApplyExpProblem) = A.state
+operator(A::ApplyExpProblem) = A.operator
+current_exponent(A::ApplyExpProblem) = A.current_exponent
+function current_time(A::ApplyExpProblem)
+  t = im*A.current_exponent
```

Comment:
```suggestion
  t = im * current_exponent(A)
```
Just a style change.